### PR TITLE
made new html page

### PR DESCRIPTION
**Pull request by HoldenDReed**

### CODE REVIEW

File: thomas.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Document</title>
+</head>
+<body>
+   <h1>This is the header to Thomas page and it is super cool and not at all cringe.</h1>
+   <p>Don't look at this paragraph Holden, if you do then I am cooler than you.</p>
```
Comment (HoldenDReed) [inline thomas.html:11]: Not true.

**Comment by kylebennett95** [comment]
Nice

File: thomas.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Document</title>
+</head>
+<body>
+   <h1>This is the header to Thomas page and it is super cool and not at all cringe.</h1>
```
Comment (kylebennett95) [inline thomas.html:10]: Cringe

**Review by kylebennett95 — COMMENTED** [review]
Cool

**Pull request merged**